pkg/cmd/cli/cmd/process.go

```diff
@@ -178,8 +178,7 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 			if len(templateName) > 0 {
 				sourceName = namespace + "/" + templateName
 			}
-			fmt.Fprintf(cmd.Out(), "unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
-			continue
+			return fmt.Errorf("unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
```

Comment:
> support of lists of templates was a side-effect of some other change and not something we need to support (see above)

Ok, then stop supporting it and take this out of a loop.  Right now the code looks incorrect.